docker/package/model.py

```diff
@@ -167,6 +167,9 @@ def gen_systemd_rules_contents(package):
     rules_contents = f"""#!/usr/bin/make -f
 
 export DEB_CFLAGS_APPEND=-fPIC
+# Disable usage of instructions from the ADX extension to avoid incompatibility
+# with old CPUs, see https://gitlab.com/dannywillems/ocaml-bls12-381/-/merge_requests/135/
+export BLST_PORTABLE=yes
```

Comment:
not sure if it makes any difference (because it's exported here), but should this also be added to `docker/package/scripts/build-binary.sh`?

Reply:
> not sure if it makes any difference (because it's exported here), but should this also be added to `docker/package/scripts/build-binary.sh`?

This script is unfortunately a part of the package's source archive, so it cannot be changed within the same upstream version.